src/components/graph/menus/switch-network-modification-active.tsx

```diff
@@ -56,11 +56,13 @@ export const SwitchNetworkModificationActive = (props: SwitchNetworkModification
     }, [modificationUuid, updateModification, setModifications]);
 
     return (
-        <Switch
-            size="small"
-            disabled={isLoading || disabled}
-            checked={modificationActivated}
-            onClick={toggleModificationActive}
-        />
+        <Tooltip title={modificationActivated ? 'Deactivate' : 'Activate'} arrow>
+            <Switch
```

Comment:
If the switch may be disabled, consider adding <span> component like noticing in the documentation here : https://mui.com/material-ui/react-tooltip/#disabled-elements

Condition to validate with PO/Design: Do we have to show tooltip if the switch is disabled ?

Otherwise, maybe add a comment to mark the code that this is a functional choice.